hack/cspell-words.txt

```diff
@@ -388,6 +393,7 @@ sendtargets
 serviceaccount
 serviceaccounts
 sgcluster
+sharded
```

Comment:
Do we have any word sharded or it is shared? I couldn't find sharded.

Reply:
sharded .. from sharding